crypto/evp_extra/p_pqdsa.c

```diff
@@ -96,16 +96,43 @@ static int pkey_pqdsa_sign_message(EVP_PKEY_CTX *ctx, uint8_t *sig,
     return 0;
   }
 
-  if (!pqdsa->method->pqdsa_sign(key->private_key, sig, sig_len, message, message_len, NULL, 0)) {
-    OPENSSL_PUT_ERROR(EVP, ERR_R_INTERNAL_ERROR);
-    return 0;
+  // |prehash| is a flag we use to indicate that the message to be signed has already
```

Comment:
Similarly to the comment above, I suggest explicitly saying something along the line
"|external_mu| indicates that the input is mu which is the result of hashing the public key, the context string and the message."

Reply:
Again, this is `PQDSA` logic, that would be true for all signature schemes in the PQDSA API, not just ML-DSA, so I don't want to add any comments that are only specific to how ML-DSA works.

Reply:
I could add a comment that says, `For ML-DSA |prehash| indicates that the input is mu which is the result of hashing the public key, the context string and the message.` or something to that effect?

Reply:
Added comment to that effect in [ba1856b](https://github.com/aws/aws-lc/pull/2113/commits/ba1856b920504afa5a4fb37ddf5ff80f70093b34)